delivery_recipe/delivery_team.md

```diff
@@ -0,0 +1,80 @@
+# Delivery Team
+
+This is a joint effort. Convivio don’t parachute into projects, do their magic then leave, like the Milk Tray Man. The best results are those delivered by a joint Convivio-client team.
+
+## Understanding the role of the team
+
+The ideal team is made up of experience, skill and authority. Experience provides us with deep, wide-ranging knowledge that we use to understand what needs to be done. The skill provides us with the tools that enable us to deliver great products. The authority clears the path of obstructions and allows “the team” to make decisions in the best interest of the business and its users.
+
+## Roles
+
+### Product Owner
+
+The Product Owner (PO) is essential to the success of a project. They represent the client’s business, its needs and they own the project. There is a huge difference between a Product Owner and a good Product Owner.
+
+A good PO understands their business, its objectives, what it needs to achieve and they have the authority to make decisions throughout the life of the change cycle.
+
+The PO’s role in the project is:
+
+* to define the requirements
+* to define the priorities
+* to provide the interface between the business and the delivery team
+* to make decisions
+* to provide detailed support to the delivery team
+
+### Service Delivery Director
+
+For long term projects or change programmes it’s important to have someone outside of the project provide a high-level bridge between the delivery team and the business.
+
+The Service Delivery Director (SDD) has experience of change and strategy to keep the client ahead of the change programme, to plan further ahead, to identify opportunities and to discuss these with shareholders. Discussing these ahead of time keeps the business one step ahead, providing shareholders with the information to make informed decisions about steering the business and aligning the different departments.
```

Comment:
shareholders should be stakeholders

Also maybe it's worth mentioning the role of coaching the product owner? And maybe worth suggesting that this is someone from Convivio while the product owner is from the client?